components/uis/BottomButton/index.tsx

```diff
@@ -4,12 +4,12 @@ import styled from "@emotion/styled";
 interface BottomButtonProps {
   disabled?: boolean;
   label: string;
-  onClick: () => void;
+  onClick?: () => void;
 }
 
 function BottomButton({ disabled = false, label, onClick }: BottomButtonProps) {
   return (
-    <StyledButton disabled={disabled} onClick={() => !disabled && onClick()}>
+    <StyledButton disabled={disabled} onClick={() => !disabled && onClick?.()}>
```

Comment:
이렇게 해도 될거 같은데 그러면 이 컴포넌트는 불필요할 거 같아

StyledButton을 바로 써도 좋을 거 같네

```suggestion
function BottomButton({ ...props}: ComponentProps<styled.button> ) {
  return (
    <StyledButton {...props} >
```

Reply:
제가 작성한 코드는 아니라서 @swimjiy 지영누나 리뷰도 부탁드림다!

Reply:
disabled 로 이미 클릭이 안돼서 굳이 없어도 되긴 해보이네요!

Reply:
[fa86933](https://github.com/mash-up-kr/musily-web/pull/85/commits/fa869336a21b1759ddb8f5e784515a4323781d7b) 으로 바꾸긴 했습니다! 이상 있으면 롤백할게요~